_includes/doc/admin-guide/options/host-override.md

```diff
@@ -3,5 +3,5 @@
 |  Type: |     string|
 | Default: ||
 
-*Description:* Replaces the ${HOST} part of the message with the
+*Description:* Replaces the HOST part of the message with the
```

Comment:
why are you removing the ${} whenthe PR and the issue states the inverse on what you did?

This way it will be hard to distinguish the actual part of the masseg from the text sorrounding it. Please consider at leas using *HOST* or `HOST`. (Same bellow)